modules/database-commons/src/main/java/org/testcontainers/ext/ScriptUtils.java

```diff
@@ -285,27 +286,47 @@ public static boolean containsSqlScriptDelimiters(String script, String delim) {
 		return false;
 	}
 
+    /**
+     * Load script from classpath and apply it to the given database
+     *
+     * @param databaseDelegate database delegate for script execution
+     * @param initScriptPath   the resource to load the init script from
+     */
+    public static void runInitScript(DatabaseDelegate databaseDelegate, String initScriptPath) {
+        runMultiInitScript(databaseDelegate, initScriptPath);
+    }
+
 	/**
-	 * Load script from classpath and apply it to the given database
+	 * Loads multiple scripts from classpath and applies it to the given database
 	 *
 	 * @param databaseDelegate database delegate for script execution
-	 * @param initScriptPath   the resource to load the init script from
+	 * @param initScriptPaths   the resources to load the init scripts from
 	 */
-	public static void runInitScript(DatabaseDelegate databaseDelegate, String initScriptPath) {
+	public static void runMultiInitScript(DatabaseDelegate databaseDelegate, String... initScriptPaths) {
```

Comment:
Do we need such big changes to `ScriptUtils` for this? It looks like [this other PR](https://github.com/testcontainers/testcontainers-java/pull/1997/files#diff-a3c7df4583fc2f224a7951d9235c8ab1R247) has a simpler approach, and I can't see why that wouldn't work...